geocoder/photon/import_from_dump.sh

```diff
@@ -12,8 +15,8 @@ sudo -E -u postgres psql postgres -c "DROP DATABASE IF EXISTS nominatim"
 
 sudo -E -u postgres psql postgres -c "CREATE DATABASE nominatim"
 
-sudo -E -u postgres psql nominatim < /nominatim_data/nominatim
+sudo -E -u postgres psql nominatim < ${HEADWAY_NOMINATIM_FILE}
 
-sudo -u photon /bin/sh -c 'java -jar photon.jar -nominatim-import -host localhost -port 5432 -database nominatim -user nominatim -password password1 -languages es,fr,de,en'
+sudo -u photon /bin/sh -c "java -jar /photon/photon.jar -nominatim-import -host localhost -port 5432 -database nominatim -user nominatim -password password1 -languages ${HEADWAY_PHOTON_LANGUAGES}"
```

Comment:
Nice, good catch. I've tried to be good about adding more than just the languages I speak but making this a build option is ideal. :)